fix(curriculum): hint and directions in RGP game step 37

sagarkashyap989 opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #52677
<!-- Feel free to add any additional description of changes below this line -->

- [x] i changed the third hint text 

- [x] i also changed the description 


this is my first open source contribution, please provide me some feedback to improve my work if it is needed

Commit:
fixed hint text and description text

huyenltnguyen reviewed (commented):
@sagarkashyap989 Thank you for the PR.

There is a change that I think we should make, but will need the Curriculum team to confirm, so let's hold off on that.

In the mean time, please help do the followings:
- Update the PR title to follow our convention: https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title
- Review the PR description and tick off the items that you have completed (I think you accidentally removed the checkboxes, and I added them back 🙂)

sagarkashyap989 commented:
> @sagarkashyap989 Thank you for the PR.
> 
> There is a change that I think we should make, but will need the Curriculum team to confirm, so let's hold off on that.
> 
> In the mean time, please help do the followings:
> 
> * Update the PR title to follow our convention: https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title
> * Review the PR description and tick off the items that you have completed (I think you accidentally removed the checkboxes, and I added them back 🙂)

- [x] updated PR title according to mentioned convention
- [x] ticked remaining checkboxes

thank you for you feedback

sagarkashyap989 commented on curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a3c2fccf186146b59c6e96.md:
````diff
@@ -15,7 +15,7 @@ You can access properties in JavaScript a couple of different ways. The first is
 ```
 
-Use dot notation to set the `onclick` property of your `button1` to the variable `goStore`. This variable will be something you write later. Note that `button1` is already declared, so you do not need to use `let` or `const`.
+Use dot notation to set the `onclick` property of your `button1` to the function reference of `goStore`. This function will be something you write later. Note that `button1` is already declared, so you do not need to use `let` or `const`.
````
> I've seen the Curriculum team updated the instructions to use contractions, so I guess we could have an improvement here as well.
> 
> Please don't make the change yet and wait for a confirmation from the team.

Thank you for pointing that out! I'll hold off on making the change for now and will wait for confirmation from the team before proceeding. Let me know if there's anything else I should consider. Appreciate your guidance

Sembauke replied:
Yep this change is fine, you can go ahead and add it.

huyenltnguyen committed:
Update curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a3c2fccf186146b59c6e96.md

gikf commented:
Hi @sagarkashyap989. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝
